client/my-sites/plans-features-main/index.jsx

```diff
@@ -50,7 +51,14 @@ class PlansFeaturesMain extends Component {
 		);
 	}
 
-	getPlanFeatures( site, intervalType ) {
+	getPlanFeatures( params ) {
+		const {
+			site,
+			intervalType,
+			upgradePlan,
+			hideFreePlan
+		} = params;
```

Comment:
Hmm, but why have you not used `this.props` when writing the method initially? Why did you opt to use params instead? I want to know what's different between having two params (and not use `this.props`) and having four (and use `this.props`). :)